tests/test_data/ad_lookup/multiple_return.json

```diff
@@ -0,0 +1,243 @@
+{
```

Comment:
do we need to consistently add all census weekend and bank holidays to all the results in this data ?

Reply:
Added all days across first 2 results, so can test for open and closed